apps/sharedtest/test/unit/dialer/font_size_manager_test.js

```diff
@@ -211,6 +225,34 @@ suite('font size manager', function() {
         });
       });
 
+      suite('on child <bdi>', function() {
+        setup(function() {
+          originalInnerHTML = document.body.innerHTML;
+          document.documentElement.style.fontSize = ROOT_FONT_SIZE + 'px';
+
+          view = document.createElement('div');
+          var bdiNode = document.createElement('bdi');
+          bdiNode.textContent = 'testing';
+          view.appendChild(bdiNode);
```

Comment:
I find the whitespace here a bit awkward. We should group all ```view``` and ```bdiNode``` operations together.